common/nvme

```diff
@@ -907,6 +907,7 @@ _find_nvme_ns() {
 	local uuid
 	local ns
 
+	udevadm settle
```

Comment:
Writing this comment after I commented the second patch.

Hmm, I think this is also not the right solution. It is changing the timing which makes it work IMO. Anyway, the fundamental problem to solve is that we need a  `_nvme_wait_for_ns` version which is able to handle these situations where the `ns` hasn't showed up yet, so some sort of retry logic with timeout as we have for the /dev/nvme0 devices.